mod-src/build.gradle.kts

```diff
@@ -130,16 +133,19 @@ task("jarAndroid") {
 		println("${reusable.size} dependencies are already desugared and can be reused.")
 		if (needReDex.isNotEmpty()) println("Desugaring ${needReDex.size} dependencies.")
 
-		// for every non-reusable dependency, invoke d8 and save the new hash
+		// for every non-reusable dependency, invoke d8 (d8.bat for windows) and save the new hash
+
+		val d8 = if(windows) "d8.bat" else "d8"
```

Comment:
there should be a space after if (not trying to pick at your commit, it just hurts me to see this)

Reply:
`if (`?

Reply:
yeah...